src/services/balancePlatform/accountHoldersApi.ts

```diff
@@ -50,9 +50,8 @@ export class AccountHoldersApi extends Service {
     /**
     * @summary Get all balance accounts of an account holder
     * @param id {@link string } The unique identifier of the account holder.
-    * @param offset {@link number } The number of items that you want to skip.
-    * @param limit {@link number } The number of items returned per page, maximum 100 items. By default, the response returns 10 items per page.
-    * @param requestOptions {@link IRequest.Options}
+    * @param requestOptions {@link IRequest.Options }
+    * @example send query parameters through requestOptions: { params: { offset: "number", limit: "number" } }
```

Comment:
Let's discuss if we want to keep it like this or make these fields, which are in the specs as well I believe, part of the function fields so it will be easier to configure. Especially as some endpoints have required query parameters.